### ISSUE TITLE
Bug: Standalone app stops responding to changes after some time

### ISSUE THREAD

**Issue by hmhealey** [mattermost/focalboard#3383]
## Steps to reproduce the behaviour

1. Have the standalone Focalboard app installed for some time.
2. Attempt to add a new card, edit an existing card, or move a card.
3. Nothing changes. If I reopen the app, I'll see my changes applied.

## Expected behaviour

The app doesn't seem to lock up randomly

## Edition and Platform

 - Edition: Personal Desktop (Mac App Store)
 - Version: 0.15.2 (33)
 - Browser and OS: MacOS 12.4

## Screenshots

<img width="313" alt="Screen Shot 2022-07-18 at 2 54 11 PM" src="https://user-images.githubusercontent.com/3277310/179582804-a97d90ad-06ea-4494-8f73-944b608d0881.png">

## Additional context (optional)

I'm not sure if this is in anyway relevant, but this version of the app still shows "Mar 2022" next to the Focalboard logo despite being a newer version. I remember mentioning this a while back, and I think this issue was known, so I'm not sure if I've gotten stuck in some weird state where the app isn't updating correctly.

**Comment by chenilim** [comment]
Thanks @hmhealey! To confirm, does closing then reopening the app solve the issue, i.e. further changes are reflected right away? Also, do you believe a specific action may have caused the refresh to break, or was this after leaving the app open for a while (e.g. overnight)?

cc @ogi-m - could you help try to repro this? Thanks.

**Comment by hmhealey** [comment]
Yeah, closing and reopening fixes the issue. I don't think there's anything specific that caused it to stop working since I've left Focalboard open overnight without any issues before. Also, I'm not 100% certain about this, but I'm pretty sure I've had this happen before in the afternoon after everything was working fine in the morning. I might be misremembering that though

**Comment by wuwinson** [comment]
@hmhealey Can you upgrade to the latest version and let us know if this is still happening? Thanks! cc @sbishel

**Comment by hmhealey** [comment]
I've got it installed via the Mac App Store, and I'm already on the latest version according to it

**Comment by hmhealey** [comment]
I tried swapping the app for the latest release on GitHub, and while that appeared to be the latest version, I lost my saved board, so I went back to the one from the App Store

**Comment by wuwinson** [comment]
Thanks @hmhealey! We haven't been able to repro this issue, so hoping that someone from the community can chime in if they're seeing something similar happen.
